.github/workflows/libclang-abi-tests.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           sed -i 's/LLVM_[0-9]\+/LLVM_NOVERSION/' $lib-${{ matrix.ref }}.abi
         done
     - name: Upload ABI file
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v3
```

Comment:
Please, commit these changes to llvm.org directly.

I missed that [https://github.com/intel/llvm/commit/7038ec5845071a60682cd66a145880994f70b995] modified a few workflows from llvm.org repository. I suggest we upstream these changes to llvm.org too to avoid merge conflicts.

Reply:
Oh... I can just revert these. I don't think we use any of these. That was automatic replacement.